pkgs/top-level/python-packages.nix

```diff
@@ -6187,6 +6187,8 @@ in {
   runway-python = callPackage ../development/python-modules/runway-python { };
 
   pyprof2calltree = callPackage ../development/python-modules/pyprof2calltree { };
+
+  opentracing = callPackage ../development/python-modules/opentracing { };
```

Comment:
I realise some may be in the wrong place but I believe this file should be in alphabetical order

Reply:
it's so damn far from alphabetical order that's not useful anyway. I recently found something like 3 or 4 potential insertion points for some new package.

Reply:
Fair enough, maybe this should be fixed at some point

Reply:
some of the packages are meant to be clumped together based on relevance, however, the current state is very from either grouping or alphabetical sorting. I think we should just have a alphabetical sorted list given that we're now at 8,000+ python packages.

Reply:
@alexarice , @multun , @jonringer Thanks for inputs. Yes, it is not very clear where to add new packages so I went for one at the end. I will try to put it back somewhere close to its group.

Reply:
I have moved this this next to opencv4